etc/systemd/tdnf-automatic-install.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=tdnf automatic install updates
+# On systems managed by either rpm-ostree/ostree, tdnf is read-only;
+# while someone might theoretically want the cache updated, in practice
+# anyone who wants that could override this via a file in /etc.
+ConditionPathExists=!/run/ostree-booted
+After=network-online.target
+
+[Service]
+Type=oneshot
+Nice=19
+IOSchedulingClass=2
+IOSchedulingPriority=7
+Environment="ABRT_IGNORE_PYTHON=1"
```

Comment:
What is the use of ABRT_IGNORE_PYTHON in tdnf automatic?

Reply:
ABRT_IGNORE_PYTHON is a python environment variable to ignore python program crashes, not needed for us, will remove it.